src/librustdoc/html/render.rs

```diff
@@ -2700,17 +2704,12 @@ fn item_trait(w: &mut Buffer, cx: &Context, it: &clean::Item, t: &clean::Trait)
             write_loading_content(w, "</div>");
         }
     }
-    write!(
-        w,
-        r#"<script type="text/javascript">window.inlined_types=new Set({});</script>"#,
-        serde_json::to_string(&synthetic_types).unwrap(),
-    );
 
     write!(
         w,
-        r#"<script type="text/javascript" async
-                         src="{root_path}/implementors/{path}/{ty}.{name}.js">
-                 </script>"#,
+        "<script type=\"text/javascript\" \
+                 src=\"{root_path}/implementors/{path}/{ty}.{name}.js\" async>\
+         </script>",
```

Comment:
Ditto this change makes it harder to read IMO.

Reply:
What was harder to read? The change itself or the generated code? If it's the latter, it's on purpose.

Reply:
I mean the change from an `r#".."#` to a `"...."` string -- which ends up with escaped quotes.

Reply:
It's same the everywhere else and it has the advantage to generate much smaller strings (which is the primary goal). At least I don't see escaped quotes as an issue.

Reply:
OK fair enough.